### ISSUE TITLE
homology calculation

### ISSUE THREAD

**Issue by huanggewudi** [drorlab/atom3d#67]
Thank you for sharing! I would like to ask how the homology segmentation in the LBA dataset is calculated. For example, I would like to calculate the homology between each protein in the test set and the training set under the 30% homology segmentation, and then sort them to see which proteins have the lowest homology for subsequent analysis.
